dev/KaairaGupta/data_loader.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis as LDA
+
+
+
+def load_split_preprocessed_data(i):
+
+    data = pd.read_csv("../../datasets/winequality.csv")
```

Comment:
Please specify the local filepath as a variable outside this call for easier local testabilitiy.

Reply:
Done!